ui/pages/setup_backup_codes.tsx

```diff
@@ -104,18 +111,84 @@ const SetupBackupCodes: NextPage = () => {
     [flow, router],
   );
 
+  let hasDisableCodes = false;
   const lookupFlow = {
     ...flow,
     ui: {
       ...flow?.ui,
-      nodes: flow?.ui.nodes.filter(({ group }) => {
-        return group === "lookup_secret";
-      }),
+      nodes: flow?.ui.nodes
+        .filter(({ group }) => {
+          return group === "lookup_secret";
+        })
+        .map((node) => {
+          if (node.meta.label?.id === ORY_LABEL_BACKUP_CODE_CREATE) {
+            node.meta.label.text = "Create backup codes";
+            node.meta.label.context = {
+              ...node.meta.label.context,
+              showBackupCodesIntro: true,
+            };
+          }
+          if (
+            node.meta.label?.id === ORY_LABEL_BACKUP_CODE_CREATE &&
+            hasDisableCodes
+          ) {
```

Comment:
nitpick, feel free to disregard: it would be nice to have boolean functions with a speaking name, instead of the boolean expression, this could help readers understand what each branch does